cli/validate.go

```diff
@@ -34,6 +35,17 @@ func validateRun(cmd *cobra.Command, args []string) {
 
 		for _, file := range files {
 			if file.IsDir() && !strings.HasPrefix(file.Name(), ".") {
+				// If the directory is empty, don't validate it
+				subfiles, err := ioutil.ReadDir(file.Name())
+				if err != nil {
+					newLogger.Critical(newCtx, "%#v", maskAny(err))
```

Comment:
Well after hitting the problem and reporting it, we exit 1. 
For me error is like: shit went down, but maybe we'll manage to fix it.
Critical: gone so south that we're all gonna die in a second.